pkg/verifier/aws/aws_verifier.go

```diff
@@ -318,3 +324,120 @@ func (a *AwsVerifier) writeDebugLogs(log string) {
 	a.Output.AddDebugLogs(log)
 	a.Logger.Debug(context.TODO(), log)
 }
+
+// CreateSecurityGroup creates a security group with the specified name and cluster tag key in a specified VPC
+func (a *AwsVerifier) CreateSecurityGroup(ctx context.Context, name, vpcId string) (*ec2.CreateSecurityGroupOutput, error) {
+	input := &ec2.CreateSecurityGroupInput{
+		GroupName:   awsTools.String(name),
+		VpcId:       &vpcId,
+		Description: awsTools.String("osd-network-verifier security group"),
+	}
+	output, err := a.AwsClient.CreateSecurityGroup(ctx, input)
+	if err != nil {
+		return nil, err
+	}
+
+	a.Logger.Info(ctx, "Created security group with ID: %s", *output.GroupId)
+
+	input_rules := &ec2.AuthorizeSecurityGroupEgressInput{
```

Comment:
This setting is likely too permissive. If possible, this should match the security group that the OpenShift installer creates

Reply:
Ah this is a least-privilege one defined by https://issues.redhat.com/browse/OSD-12356, however we definitely need to tag this security group with `"red-hat-managed: true"`

* No ingress allowed
* Allowed egress to 0.0.0.0/0 for ports 80/443/9997 since we can't restrict egress by domain name